docs/src/development/local/lando.md

```diff
@@ -33,54 +33,60 @@ To use a build hook that requires environment variables, manually [add them](htt
 
 Follow the [Lando installation instructions](https://docs.lando.dev/getting-started/installation.html).
 
-## 2. Initialize Lando
+## 2. Create an access token
+
+To authorize Lando to communicate with Platform.sh, create an [API token](../../administration/cli/api-tokens.md#get-a-token).
+Copy the value.
+
+## 3. Initialize Lando
 
 {{< codetabs >}}
 
 ---
-title=On a new Platform.sh project without code
+title=On an existing Platform.sh project
 file=none
 highlight=false
 ---
 
-For a quicker start, create a project based on the Platform.sh [PHP template](https://github.com/platformsh-templates/php).
-The template provides the most basic configuration for running a custom PHP project built with Composer.
-It also includes the required Platform.sh configuration files out of the box.
+If your code isn't present locally, retrieve your codebase with one of these methods:
 
-1. [Create a new project based on the PHP template]({{% create-project-link template="php" %}}).
-2. Clone that project locally in one of these ways:
-    - Using the [Platform.sh CLI](../../gettingstarted/introduction/own-code/cli-install.md) by running `platform get <PROJECT_ID>`
-    - Using [git](../../administration/web/configure-environment.md#actions)
-3. In the project's folder, run `lando init --recipe platformsh --source platformsh`.
-4. Follow the instructions provided by the interactive prompt.
-   On the `Which project?` step, select the project created in the first step.
+- Using the [Platform.sh CLI](../../administration/cli/_index.md) by running `platform get <PROJECT_ID>`
+- Using [Git](../../administration/web/configure-environment.md#actions)
+
+Otherwise, access the directory with your project.
+
+Run `lando init --recipe platformsh --source cwd --platformsh-auth <API_TOKEN>` and follow the instructions provided by the interactive prompt.
```

Comment:
I have the Platform.sh CLI installed and I still was requested to add a token. It also happened in the original issue. It seems like being explicit in adding it can be helpful.

I don't think that the `adapt and` phrasing helps so much.

Reply:
Okey sure, then let's leave it that way :)